src/test/regress/multi_mx_schedule

```diff
@@ -34,7 +34,6 @@ test: multi_mx_tpch_query7_nested multi_mx_ddl
 test: ch_bench_having_mx
 test: recursive_dml_queries_mx multi_mx_truncate_from_worker
 test: multi_mx_repartition_udt_prepare mx_foreign_key_to_reference_table
-test: multi_mx_repartition_join_w1 multi_mx_repartition_join_w2 multi_mx_repartition_udt_w1 multi_mx_repartition_udt_w2
```

Comment:
do we need to remove these?

Reply:
initially I didn't remove them but we were sometimes getting too many clients error even with pool size = 2.

Then I looked at the files and as they were simple(similar to other repartition joins) I removed them. 

If you think we shouldn't remove them, we can try to have them 2 by 2 instead of 4 of them running concurrently.

Reply:
These 4 tests are running concurrently for a specific reason: https://github.com/citusdata/citus/blob/9229db2081ee8f53132afc2a407f0be23db6ba96/src/test/regress/sql/multi_mx_repartition_join_w1.sql#L1-L6

So, I think we should not remove them and keep running concurrently.

Reply:
@onderkalaci do you have any suggestion on how not to get "too many clients" error with all running concurrently?

Reply:
That sounds like a bug. I would not expect more than ~8 connections per worker per query if the max_adaptive_executor_pool_size is 2 (1 client connection, 2 executor connections, and 4 worker-to-worker connections for data fetches, maybe 1 for creating/dropping job scheams).

Reply:
enabled debug logs: https://circleci.com/gh/citusdata/citus/136375?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

will investigate why this is the case ( It could be that we are not closing the connections when we should)

Reply:
https://github.com/citusdata/citus/issues/3971

Reply:
bc69de9